testgrid/config.yaml

```diff
@@ -4314,6 +4317,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-scalability-stable1
   - name: gci-gce-scalability-1.11
     test_group_name: ci-kubernetes-e2e-gci-gce-scalability-stable2
+  - name: gci-gce-scalability-1.10
```

Comment:
I was always confused with that (there is also beta and latest - I think latest is master, and eta is now 1.13).
@krzyzacy @BenTheElder @cjwagner - can one of you confirm?

@shyamjvs - re renaming, I can do that, but I'm OOO this week, so if we could do that in follow up, it would be great.

Reply:
@shyamjvs I can simply send a follow up PR for now given Wojtek is OOO I guess

Reply:
SG.